Plugins/Json/include/Acts/Plugins/Json/JsonHierarchicalObjectConverter.hpp

```diff
@@ -0,0 +1,183 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2017-2019 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include <Acts/Geometry/TrackingVolume.hpp>
+#include <Acts/Surfaces/Surface.hpp>
+#include "Acts/Geometry/HierarchicalGeometryContainer.hpp"
+#include "Acts/Geometry/TrackingGeometry.hpp"
+#include "Acts/Utilities/Definitions.hpp"
+#include "Acts/Utilities/Logger.hpp"
+
+#include <nlohmann/json.hpp>
+
+#include <map>
+
+namespace Acts {
+
+/// @class JsonHierarchicalObjectConverter
```

Comment:
Neither the `@class` line nor the `@brief` tag are needed. Doxygen picks that up automatically.

Reply:
I didn't know thanks for the info. It has be changed